guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
<blockquote>The PSPNet model is not a complete segmentation model in itself, it is just an encoder, which means it is just half of what is required for image segmentation. The most common decoders that are found in various implementations of PSPNet is a convolution layer followed by a 8x bilinear-upsampling.</blockquote><p> </p><p>can we have a decoder definition before this para?</p><p>&lt;this is what a decoder is...........&gt;</p><p> </p><p>Note: PSPNet model is not a complete segmentation model in itself. It is just an encoder, which means it is just half of what is required for image segmentation. <strong>The other half is _________&lt;decoders?&gt;_</strong> </p><p>The most common decoders that are found in various implementations of PSPNet is a convolution layer followed by a 8x bilinear-upsampling.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Done.</p>